### PR TITLE
Update pipelines to new build ADO references

### PR DESCRIPTION
**Pull request by thomast1906**
### Change description ###
Update pipelines to new build ADO references

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: azure-pipelines.yaml [hunk 1]
```diff
@@ -41,23 +41,23 @@ parameters:
...
         timeForRed: '7 days'
         pipelineName: "ACR Base importer"
         branchName: "refs/heads/master"
       - project: 'Platform%20Engineering'
-        definitionId: '224'
+        definitionId: '756'
         timeForAmber: '3 days'
         timeForRed: '5 days'
-        pipelineName: "Preview scheduled cleanup"
+        pipelineName: "AKS Preview Helm Release Cleanup job"
```
Comment (adusumillipraveen) [inline azure-pipelines.yaml:60]:
```suggestion
        pipelineName: "AKS Preview Helm Release Scheduled Cleanup job"
```

Comment (thomast1906) [reply azure-pipelines.yaml:60]: I dont see that pipeline name in ADO currently?

**Review by adusumillipraveen — APPROVED** [review]

File: azure-pipelines.yaml [hunk 2]
```diff
@@ -17,20 +17,20 @@ parameters:
...
     type: object
     default:
-      - project: 'cnp'
-        definitionId: '37'
+      - project: 'PlatformOperations'
+        definitionId: '802'
         timeForAmber: '3 days'
         timeForRed: '5 days'
-        pipelineName: "Preview PR cleanup"
+        pipelineName: "AKS Preview Helm Release Cleanup job"
```
Comment (adusumillipraveen) [inline azure-pipelines.yaml:24]:
```suggestion
        pipelineName: "AKS Preview Helm Release Cleanup"
```

File: azure-pipelines.yaml [hunk 3]
```diff
@@ -41,23 +41,23 @@ parameters:
...
         pipelineName: "ACR Base importer"
         branchName: "refs/heads/master"
-      - project: 'Platform%20Engineering'
-        definitionId: '224'
+      - project: 'PlatformOperations'
+        definitionId: '759'
         timeForAmber: '3 days'
         timeForRed: '5 days'
-        pipelineName: "Preview scheduled cleanup"
+        pipelineName: "AKS Cleanup NONPROD"
```
Comment (adusumillipraveen) [inline azure-pipelines.yaml:60]:
```suggestion
        pipelineName: "Preview Helm release cleanup - PR close"
```

File: azure-pipelines.yaml [hunk 4]
```diff
@@ -17,20 +17,20 @@ parameters:
...
     type: object
     default:
-      - project: 'cnp'
-        definitionId: '37'
+      - project: 'PlatformOperations'
+        definitionId: '802'
         timeForAmber: '3 days'
         timeForRed: '5 days'
-        pipelineName: "Preview PR cleanup"
+        pipelineName: "AKS Preview Helm Release Cleanup"
```
Comment (adusumillipraveen) [inline azure-pipelines.yaml:24]:
```suggestion
        pipelineName: "Preview Helm release cleanup - scheduled"
```

**Review by timja — APPROVED** [review]

**Pull request merged**